input/pagecontent/changes.md

```diff
@@ -14,4 +14,5 @@ This change log documents the significant updates and resolutions implemented fr
   - made ServiceRequest.encounter mandatory (1..1) [FHIR-47008](https://jira.hl7.org/browse/FHIR-47008)
   - added Must Support to ServiceRequest.priority [FHIR-46939](https://jira.hl7.org/browse/FHIR-46939)
   - AU eRequesting ServiceRequest profile renamed to be AU eRequesting Diagnostic Request [FHIR-46842](https://jira.hl7.org/browse/FHIR-46842)
+  - Added new ValueSet [AU eRequesting Request Status](ValueSet-au-erequesting-request-status.html) [FHIR-47097](https://jira.hl7.org/browse/FHIR-47097)
```

Comment:
Please change to something like: 
Removed draft and unknown from ServiceRequest.status [FHIR-47097](https://jira.hl7.org/browse/FHIR-47097)

Reply:
Change description updated as requested.